content/html/aarch64-request-hosting.html

```diff
@@ -108,10 +108,12 @@ <h4>OpenStack Access Question(s)</h4>
         </label>
         <select id="edit-submitted-flavor" name="flavor" class="form-select required">
           <option value="None selected" selected="selected">- Select -</option>
-          <option value="m1_small">1 CPU, 2G RAM, 20G Disk</option>
-          <option value="m1_medium">2 CPU, 4G RAM, 40G Disk</option>
-          <option value="m1_large">4 CPU, 8G RAM, 80G Disk</option>
-          <option value="m1_xlarge">8 CPU, 16G RAM, 160G Disk</option>
+          <option value="tiny">1 CPU, 1G RAM, 4G Disk</option>
+          <option value="small">1 CPU, 2G RAM, 20G Disk</option>
+          <option value="medium">2 CPU, 4G RAM, 40G Disk</option>
+          <option value="large">4 CPU, 8G RAM, 80G Disk</option>
+          <option value="xlarge">8 CPU, 16G RAM, 160G Disk</option>
+          <option value="xxlarge">16 CPU, 32G RAM, 320G Disk</option>
```

Comment:
For whatever reason, these seem to be different on this cluster. This actually matches what's used on the system currently. We probably should change that eventually but let's make sure this aligns properly.
```suggestion
          <option value="tiny">1 CPU, 512M RAM, 4G Disk</option>
          <option value="small">1 CPU, 1G RAM, 15G Disk</option>
          <option value="medium">2 CPU, 4G RAM, 30G Disk</option>
          <option value="large">4 CPU, 8G RAM, 60G Disk</option>
          <option value="xlarge">8 CPU, 16G RAM, 80G Disk</option>
          <option value="xxlarge">16 CPU, 32G RAM, 320G Disk</option>
```